### PR TITLE
Add msdocs-django-postgresql-sample-app-azd to gallery

### PR DESCRIPTION
**Pull request by hemarina**
Add template based on #61 
Open this PR for reviews. Will change author to Azure-samples once it's approved.

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -229,7 +229,16 @@ const Users: User[] = [
...
   //-------  ALL OTHER TEMPLATES go here, can be searched for by name
-
+  {
+    title: 'Python (Django) Web App with PostGreSQL',
+    description:
+      'A restaurants review web app that uses whitenoise for static assets and PostGreSQL for the database. Ready for deployment on App Service with a PostGreSQL Flexible Server inside a VNet.',
+      preview: require('./images/msdocs-django-postgresql-sample-app-azd.png'),
+    website: 'https://github.com/pamelafox',
+    author: 'Pamela Fox',
+    source: 'https://github.com/pamelafox/msdocs-django-postgresql-sample-app-azd',
```
Comment (jongio) [inline website/src/data/users.tsx]: I think @pamelafox was going to move this to azure-samples?

Comment (pamelafox) [reply website/src/data/users.tsx]: Yes, still waiting for ghrnotify@ to unlock the forked repository (they automatically lock all forks of repos within the same org, as it's often a mistake, but I believe ours is a legitimate use).

Comment (pamelafox) [reply website/src/data/users.tsx]: The content team decided they're comfortable with the changes and would rather not have forks lying around, so the changes are now in the official Azure-samples repo:
https://github.com/Azure-Samples/msdocs-django-postgresql-sample-app

Not sure what the author should be in this case? I authored the azd changes but it's now hosted in Azure-samples and the original author is the content team.

Comment (hemarina) [reply website/src/data/users.tsx]: Based on existing templates, if it's in azure-samples, author is Azure Dev. Do you have suggestions or thoughts on author? @jongio

Comment (jongio) [reply website/src/data/users.tsx]: "Azure Content Team" as author is fine.

Comment (pamelafox) [reply website/src/data/users.tsx]: Sounds good, these changes should be ready to make now, by the way.

**Review by pamelafox — APPROVED** [review]
Looks correct to me

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -229,7 +229,16 @@ const Users: User[] = [
   },
 
   //-------  ALL OTHER TEMPLATES go here, can be searched for by name
-
+  {
+    title: 'Python (Django) Web App with PostGreSQL',
```
Comment (savannahostrowski) [inline website/src/data/users.tsx]: Should be "PostgreSQL"

File: website/src/data/users.tsx [hunk 3]
```diff
@@ -229,7 +229,16 @@ const Users: User[] = [
...
+    description:
+      'A restaurants review web app that uses whitenoise for static assets and PostGreSQL for the database. Ready for deployment on App Service with a PostGreSQL Flexible Server inside a VNet.',
```
Comment (savannahostrowski) [inline website/src/data/users.tsx]: Same correction here!

**Review by savannahostrowski — APPROVED** [review]
Minor comment but otherwise LGTM! 🚀

File: website/src/data/users.tsx (hunk 2 above)
Comment (pamelafox) [reply website/src/data/users.tsx]: Ooo, I didn't realize that was the correct capitalization. I need to go fix it in a bunch of my own stuff now. :)

**Pull request merged**